docker-compose.yml

```diff
@@ -55,6 +55,7 @@ services:
     extends: base
     command: flask rq worker --worker-ttl 450
     environment:
+      - SERVER_NAME=app
```

Comment:
I don't think this is needed … was something not working in your setup without it?

Reply:
Yes the nomad_worker was erroring out.